### PR TITLE
Remove checks related to no longer supported Vagrant versions

### PR DESCRIPTION
**Pull request by wbclark**
These checks were required in CI back in 2017. Since we
only support Vagrant 2.2+ presently, they can be safely
removed.

### CODE REVIEW

File: vagrant/lib/forklift/box_distributor.rb [hunk 1]
```diff
@@ -184,12 +179,8 @@ def configure_ansible(machine, ansible, box_name)
...
       [playbooks].flatten.each_with_index do |playbook, index|
-        if SUPPORT_NAMED_PROVISIONERS
-          args = ["main#{index}"]
-          kwargs = { type: 'ansible' }
-        else
-          args = [:ansible]
-        end
+        args = ["main#{index}"]
+        kwargs = { type: 'ansible' }
         machine.vm.provision(*args, **kwargs) do |ansible_provisioner|
```
Comment (ekohl) [inline vagrant/lib/forklift/box_distributor.rb:184]:
```suggestion
        machine.vm.provision("main#{index}", type: 'ansible') do |ansible_provisioner|
```

**Review by ekohl — APPROVED** [review]
I recall last time we had issues with Vagrant on EL7, mostly regarding downstream. I'm not sure those are still an issue. @ehelms knew last time.

I'd also be fine merging this and letting it break.

**Review by ehelms — APPROVED** [review]

**Comment by evgeni** [comment]
Even on EL7 we deploy 2.2.x:

https://github.com/theforeman/forklift/blob/5683c94fb95fca963f98512d5094c4b7e964c52d/roles/vagrant/defaults/main.yml#L3

File: vagrant/lib/forklift/box_distributor.rb (hunk 1 above)
Comment (wbclark) [reply vagrant/lib/forklift/box_distributor.rb]: TY

**Comment by wbclark** [comment]
If someone would kindly merge this, I do not have the permissions to do so

**Pull request merged**